### PR TITLE
docs: add GreptimeDB Enterprise

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

image

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/enterprise/overview.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Overview
+
+GreptimeDB Enterprise is a robust time-series database solution tailored to meet the specific needs of enterprises.
+With customized solutions and 24/7 rapid technical support,
+GreptimeDB Enterprise helps businesses optimize data efficiency and significantly reduce costs to help you make smarter, faster decisions using time-series data.
+
```
Comment (nicecui) [inline docs/enterprise/overview.md:6]: PTAL

**Review by fengjiachun — APPROVED** [review]

File: docs/enterprise/overview.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Overview
+
+GreptimeDB Enterprise is a robust time-series database solution tailored to meet the specific needs of enterprises.
+With customized solutions and 24/7 rapid technical support,
```
Comment (sunng87) [inline docs/enterprise/overview.md:4]: We should not guarantee service level in tech docs.

**Comment by nicecui** [comment]
@sunng87 PTAL

File: docs/enterprise/overview.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Overview
```
Comment (nicecui) [inline docs/enterprise/overview.md:1]: @sunng87 PTAL

**Review by sunng87 — APPROVED** [review]

**Pull request merged**